events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* Reviewing comms written by other shadows, volunteers, or the lead as directed by the KCS Comms Lead
+
+Shadows should work with the Comms Lead to ensure they are getting sufficient experience to feel comfortable taking on the KCS Comms Lead role in a future cycle after completion of a shadow cycle.
+
+In the event a shadow is unable to complete the tasks expected of them as designated by the KCS Comms Lead, they should inform the KCS Comms Lead as early as possible, to the best of their ability. Shadows who are unable to complete the required tasks as designated by the KCS Comms Lead, will be removed from the list of shadows. Upon failure to complete a shadowing cycle, the contributor may request to become a shadow in a future cycle. Successful completion of a shadowing cycle (as determined by the KCS Comms Lead) or equivalent experience is required to become a KCS Comms Lead.
+
+## KCS Comms Volunteer Overview
+Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].
+
+### Requirements
+* Review this document.
+* Commit to supporting KCS Comms through the current cycle
+
+### Tasks
+During the course of their KCS cycle as a volunteer, KCS Comms Volunteers are expected to complete tasks including but not limited to:
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* [optional] Reviewing comms written by other volunteers, shadows, or the lead as directed by the KCS Comms Lead
+
+## KCS Website Manager
+The KCS Website Manager is expected to collaborate with all KCS Staff leads and their teams to understand what changes need to be made to the website, when those changes need to go live, and ensuring that all KCS-related website changes are planned, prioritzed, and completed.
+
+### Requirements
+- Review this document.
+- [Kubernetes Org Membership]
+- Commit to supporting KCS Comms through the current cycle and the following cycle as emeritus Website Manager (1 year, 2 cycles)
+- Basic familiarity with the [Kubernetes Contributor Site (k8s.dev)](https://github.com/kubernetes/contributor-site), especially the events section (k8s.dev/events)
+- Basic familiarity with the [Hugo website framework](https://gohugo.io/)
+- Basic familiarity with the [Markdown](https://www.markdownguide.org/) markup language
+
+### Tasks
+During the course of their KCS cycle as a website manager, KCS Website Managers are expected to complete tasks including but not limited to:
```

Comment:
Can there be multiple website managers (the term webmaster from the 90s did pop into my head, but that's only because I was one in high school).

Reply:
Theoretically, yes, but in practice we've struggled to recruit even one.